test/helpers/DetailedERC20Mock.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.4.11;
+
+import '../../contracts/token/StandardToken.sol';
+import '../../contracts/token/DetailedERC20.sol';
+
+contract DetailedERC20Mock is StandardToken, DetailedERC20 {
+  function DetailedERC20Mock(uint256 _decimals, string _symbol, string _name) {
```

Comment:
I think I'd actually make this the constructor of `DetailedERC20` itself. In that way, contracts can inherit from it and call the constructor without having to modify the state variables. It will look cleaner IMO. (And the mock wouldn't be necessary for testing.)

Also, the order of arguments `(name, symbol, decimals)` makes a bit more sense to me.